packages/mobile/src/fiatExchanges/FiatExchangeOptions.tsx

```diff
@@ -18,7 +18,7 @@ import { KOTANI_URI, PONTO_URI, SIMPLEX_URI } from 'src/config'
 import FundingEducationDialog from 'src/fiatExchanges/FundingEducationDialog'
 import i18n, { Namespaces } from 'src/i18n'
 import InfoIcon from 'src/icons/InfoIcon'
-import RadioIcon from 'src/icons/RadioIcon'
+import RadioButton from 'src/icons/RadioButton'
```

Comment:
Importing from the wrong place. Should be `@celo/react-components/components/RadioButton`. Looking at the file path for the file you created ;)